php/libraries/NDB_Caller.class.inc

```diff
@@ -128,6 +128,22 @@ class NDB_Caller
                 $p = $mname;
             }
             $this->page = $m->LoadPage($p);
+
+            // FIXME: These two method_exists calls were hacks that were previously
+            // in the load* classes, which for now need to be preserved for
+            // compatibility with existing modules.
+            //
+            // The proper solution is probably to re-write anything that uses them
+            // in React.
+            if (method_exists($this->page, 'getControlPanel')) {
+            $this->controlPanel = $this->page->getControlPanel();
+            }
+            if (method_exists($this->page, 'getFeedbackPanel')) {
+                $this->feedbackPanel = $this->page->getFeedbackPanel(
+                    $_REQUEST['candID'],
+                    $_REQUEST['sessionID'] ?? null
+                );
+            }
```

Comment:
This is the same as/copy/pasted from #3142, one of them will have conflicts and need to be rebased, but it's necessary for the control panel to work..